### DISCUSSION TITLE
Problem installing PiMox HAOS VM script in CM3588 (ARM64)

### DISCUSSION THREAD

**Discussion by Pharizna** [tteck/Proxmox#3335] [Help Needed]
By mistake (ignorance) I asked [this question](https://forum.proxmox.com/threads/problem-installing-pimox-haos-vm-script-in-cm3588-arm64.150730/#post-681571) in the PROXMOX Forum where I was "kicked out" in a very impolite way :( I'm sure I'll be treated better here..

"Hi and thank in advance for your help.

I've just installed Proxmox on Debian 12 in my new **CM3588 NAS KIT device (ARM64)** using this [article](https://github.com/jiangcuo/Proxmox-Port/wiki/Install-Proxmox-VE-on-Debian-%20bookworm).

It seems Proxmox is running fine but I can't use any of[ these scripts](https://tteck.github.io/Proxmox/#pimox-haos-vm),

For example, in the PiMox Home Assistant one, it seems that the VM creation is fine... but the Home Assistant OS can't start.

✓ Created HAOS VM (haos12.4)
- Starting Home Assistant OS VM...KVM virtualisation configured, but not available. Either disable in VM configuration or enable in BIOS.
‼ ERROR 255@322 Unknown failure occurred.
![error](https://github.com/tteck/Proxmox/assets/3939655/48af8613-c17d-467a-905c-ed562eef88f3)

In other cases, such as PiHole for example, there is simply a message of "This script will not work with PiMox!" even though [apparently](https://github.com/asylumexp/Proxmox) it should be compatible, right?

Help me, please :)

Pedro

**Comment by tteck** [comment]
Is that device on the supported [device list](https://github.com/jiangcuo/Proxmox-Port/wiki/Support-List)?

No scripts in this repository (other than PiMox HAOS VM) support ARM64.

**Comment by tteck** [comment]
Looks like your device doesn't support virtualization. 🤷🏻‍♂️

**Reply by Pharizna** [reply]
THANK YOU for your help. It's a pity :(

**Reply by Pharizna** [reply]
It's very strange because OrangePI 5/5 plus has the same processor (Rokchip RK3588) as the CM3588 NAS KIT. Isn't?
